src/oop/user_defined_classes/medium/q4/QuestionText.md

```diff
@@ -0,0 +1,19 @@
+# oop-user_defined_classes-medium-q4
+
+## Java
+
+We always get thirsty in the summer and it is important to stay hydrated. Make a Java 
+class called Water that has two attributes called volume and quality with a constructor 
+that initializes them.
+
+
+
+## Water
+
+The `Water` class you are given has the following items:
+
+- Attributes:
+  - A `double` called **volume**
```

Comment:
For this one, we should also add a check to the constructor to see if the volume is not negative.